src/libopensc/sc-ossl-compat.h

```diff
@@ -114,208 +55,11 @@ extern "C" {
 #define EVP_PKEY_get_raw_public_key(p, pu, l)   (0)
 #endif
 
+/* OpenSSL 1.1.1 has FIPS_mode function */
 #if OPENSSL_VERSION_NUMBER >= 0x30000000L
-#define EC_POINT_get_affine_coordinates_GFp     EC_POINT_get_affine_coordinates
-#define EC_POINT_set_affine_coordinates_GFp     EC_POINT_set_affine_coordinates
```

Comment:
These two are also deprecated in OpenSSL 3.0 so we need to keep this defines.

Reply:
This PR changed these other files too:
modified: sc-ossl-compat.h
modified: ../pkcs11/framework-pkcs15.c
modified: ../pkcs11/openssl.c
modified: ../pkcs15init/pkcs15-westcos.c
modified: ../tools/piv-tool.c
modified: ../tools/pkcs15-init.c

EC_POINT_set_affine_coordinates_GFp   was changed to  EC_POINT_set_affine_coordinates in the few places that used it. 

And there is a pdf:  https://github.com/OpenSC/OpenSC/files/7987846/sc-ossl-compat.pdf in the original conversation of a spread sheet that listed  how some of these old names that have changed.  

The PR has also assumed that we are only going to support OpenSC 1.1.1 and 3.0.0 and above. And also support LibreSSL 3.4.2 (or so) and above. So many of the old names have changed. Look at the PDF